rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringNodeView.java

```diff
@@ -180,7 +180,8 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        status.setContents("Monitoring is unavailable on this node. Maybe node was started with --disableMonitoring option.<br>"+
+                "Error message: "+ JSONUtils.getJsonErrorMessage(caught));
```

Comment:
Monitoring is unavailable on this node. Either monitoring was disabled with the disableMonitoring option or an error occurred. More information are available inside the node logs.